Headers cli flag

bconnorwhite opened the pull request:
Added option to set headers through cli tool

zhigang1992 commented:
Can you add doc to readme? also would it be better to do `-h 'Authorization: Bear abcd'` instead. Better for commandline.

bconnorwhite commented:
I forgot you can use the same flag multiple times with commander, I've adjusted this now to stack multiple headers:
`... -H 'Authorization: Bearer abcd' -H 'Another: header'`

Also added the header option to the README. I left the short flag as `-H` instead of `-h` though to avoid a collision with the `-h` help flag, but feel free to change this

zhigang1992 commented:
@helios1138 plz take a look. 😄
